src/dinit-iostream.cc

```diff
@@ -0,0 +1,623 @@
+#include <algorithm>
+#include <exception>
+#include <cstring>
+
+#include <dinit-iostream.h>
+#include <baseproc-sys.h>
+
+// See dinit-iostream.h for inforamtion about the usage
+
+// A simple function for calculating how much digits can be in a type
+// TODO: Putting it in dinit-util.h? Maybe
```

Comment:
I don't know it should be there or not. WDYT? (sorry, It's first and last TODO in this code)